README.md

```diff
@@ -32,56 +35,42 @@ Read [release notes](https://github.com/openvinotoolkit/model_server/releases) t
-- Support for [AI accelerators](https://docs.openvino.ai/nightly/about-openvino/compatibility-and-support/supported-devices.html)
+- Support for [AI accelerators](./docs/accelerators.md)
 
-**Note:** OVMS has been tested on RedHat, and Ubuntu. The latest publicly released docker images are based on Ubuntu and UBI.
-They are stored in:
+**Note:** OVMS has been tested on RedHat, Ubuntu and Windows. 
+Public docker images are stored in:
 - [Dockerhub](https://hub.docker.com/r/openvino/model_server)
 - [RedHat Ecosystem Catalog](https://catalog.redhat.com/software/containers/intel/openvino-model-server/607833052937385fc98515de)
-
+Binary packages for Linux and Windows are on [Github](https://github.com/openvinotoolkit/model_server/releases)
 
 ## Run OpenVINO Model Server
 
-A demonstration on how to use OpenVINO Model Server can be found in our quick-start guide [for vision use case](docs/ovms_quickstart.md) and [LLM text generation](docs/llm/quickstart.md).
-For more information on using Model Server in various scenarios you can check the following guides:
-
-* [Model repository configuration](docs/models_repository.md)
-
-* [Deployment options](docs/deploying_server.md)
+A demonstration on how to use OpenVINO Model Server can be found in our [quick-start guide for vision use case](docs/ovms_quickstart.md) and [LLM text generation](docs/llm/quickstart.md).
 
-* [Performance tuning](docs/performance_tuning.md)
+Check also other instructions:
 
-* [Directed Acyclic Graph Scheduler](docs/dag_scheduler.md)
+[Preparing model repository](https://docs.openvino.ai/nightly/openvino-workflow/model-server/ovms_docs_models_repository.html)
 
-* [Custom nodes development](docs/custom_node_development.md)
+[Deployment](https://docs.openvino.ai/nightly/openvino-workflow/model-server/ovms_docs_deploying_server.html)
 
-* [Serving stateful models](docs/stateful_models.md)
+[Writing client code](https://docs.openvino.ai/nightly/openvino-workflow/model-server/ovms_docs_server_app.html)
 
-* [Deploy using a Kubernetes Helm Chart](https://github.com/openvinotoolkit/operator/tree/main/helm-charts/ovms)
-
-* [Deployment using Kubernetes Operator](https://operatorhub.io/operator/ovms-operator)
-
-* [Using binary input data](docs/binary_input.md)
+[Demos](https://docs.openvino.ai/nightly/openvino-workflow/model-server/ovms_docs_demos.html)
 
 
 
 ## References
 
 * [OpenVINO&trade;](https://software.intel.com/en-us/openvino-toolkit)
 
-* [TensorFlow Serving](https://github.com/tensorflow/serving)
+* [RAG building blocks made easy and affordable with OpenVINO Model Server](https://medium.com/openvino-toolkit/rag-building-blocks-made-easy-and-affordable-with-openvino-model-server-e7b03da5012b)
 
-* [gRPC](https://grpc.io/)
+* [Simplified Deployments with OpenVINO™ Model Server and TensorFlow Serving](https://community.intel.com/t5/Blogs/Tech-Innovation/Artificial-Intelligence-AI/Simplified-Deployments-with-OpenVINO-Model-Server-and-TensorFlow/post/1353218)
```

Comment:
I suppose this blog introduces `ovmsclient` that is no longer recommended way to prepare a client. Perhaps we should drop this reference?